src/Octoshift/GithubApi.cs

```diff
@@ -581,5 +558,51 @@ public virtual async Task<MannequinReclaimResult> ReclaimMannequin(string orgId,
 
             return data.ToObject<MannequinReclaimResult>();
         }
+
+        private static object GetMannequinsPayload(string orgId)
+        {
+            var query = "query($id: ID!, $first: Int, $after: String)";
+            var gql = @"
+                node(id: $id) {
+                    ... on Organization {
+                        mannequins(first: $first, after: $after) {
+                            pageInfo {
+                                endCursor
+                                hasNextPage
+                            }
+                            nodes {
+                                login
+                                id
+                                claimant {
+                                    login
+                                    id
+                                }
+                            }
+                        }
+                    }
+                }";
+
+            return new
+            {
+                query = $"{query} {{ {gql} }}",
+                variables = new { id = orgId }
+            };
+        }
+
+        private static Mannequin MannequinBuilder(JToken mannequin)
```

Comment:
nit: a small naming refactor to comply with MS naming guidelines --> `BuildMannequin`

Reply:
done